lib/tests/resources/tasks-cache/8fdd86d9be77a27b296d5b6f2d2d984fd7d671bc8bb1ffc86f6c6b0492ec4d01/init.bat

```diff
@@ -0,0 +1 @@
+type %PT__installdir\file1.txt% %PT__installdir\dir\file2.txt% %PT__installdir\dir\sub_dir\file3.txt% 2>nul
```

Comment:
Oh, the problem here is that an environment variable in bat is accessed with `%envvar%`. So it should be `type %PT__installdir%\file1.txt`.

Reply:
That is one problem. The error message in appveyor shows the expanded paths, it just does not appear to be printing the file contents cleanly. Will keep looking at this, trying `@echo OFF` on this iteration.